bundles/tampere/content-editor/view/SideContentEditor.js

```diff
@@ -497,22 +647,70 @@ Oskari.clazz.define('Oskari.tampere.bundle.content-editor.view.SideContentEditor
          * @param  {Boolean}    deleteFeature is delete feature
          */
         sendRequest: function (requestData, deleteFeature) {
+            var overlay = Oskari.clazz.create('Oskari.userinterface.component.Overlay');
+            overlay.overlay('body');
+            overlay.followResizing(true);
+            var spinner = Oskari.clazz.create('Oskari.userinterface.component.ProgressSpinner');
+            spinner.insertTo('body');
+            spinner.start();
+
             var me = this,
                 okButton = Oskari.clazz.create('Oskari.userinterface.component.Button'),
                 url = null,
                 wfsLayerPlugin = me.sandbox.findRegisteredModuleInstance('MainMapModule').getPluginInstances('WfsLayerPlugin');
 
             okButton.setTitle(me.loc.buttons.ok);
-            if (me.operationMode === 'create') {
+            var multipleFeaturesData = [];
+
+            if (me.operationMode === 'create' && !me.editMultipleFeatures) {
                 url = Oskari.urls.getRoute('InsertFeature');
-            } else {
+            } else if(!me.editMultipleFeatures) {
+                url = Oskari.urls.getRoute('SaveFeature');
+            } else if(me.editMultipleFeatures) {
                 url = Oskari.urls.getRoute('SaveFeature');
+                var elements = jQuery('.getinforesult_table');
```

Comment:
Could you just pass the payload to be sent to the server as a parameter instead of building it in this method?